R/topology.R

```diff
@@ -774,7 +774,11 @@ graph.isoclass <- isoclass_impl
 #' @param directed Whether to create a directed graph (the default).
 #' @return An igraph object, the graph of the given size, directedness
 #'   and isomorphism class.
-#'
+#' @return A list with components:
+#'   \describe{
+#'     \item{topology}{The topology of the graph.}
+#'     \item{details}{Additional details about the topology.}
+#'   }
```

Comment:
```suggestion
```

THis return value does not make any sense. There is only a graph returned as written in the return doc above